R/palettes.R

```diff
@@ -0,0 +1,43 @@
+## This color palettes will be used by default for plotting outputs. They take a
```

Comment:
Somehow I feel uneasy about these being loose variables.

Can you (for now at least) put these into a function that returns a list, and elsewhere call that function?  So

```r
odin_ui_palettes <- function() {
  list(brewer_set1 = c(...),
        ...)
}
```

Then elsewhere you can do `choices <- names(odin_ui_palettes())` and the `get` is replaced by a lookup.

Then we can later extend this to allow additional palettes to be added